weather.py

```diff
@@ -90,7 +132,39 @@ def generate_summary(weather_data):
     Returns:
         A string containing the summary information.
     """
-    pass
+    sum_min = 0
+    sum_max = 0
+    total_items = len(weather_data)
+    min_value = (((weather_data[0][1])-32)*5)/9
+    max_value = (((weather_data[0][2])-32)*5)/9
```

Comment:
Hint: you created some functions earlier (e.g. `convert_f_to_c`) which could be useful to use here :)